packages/api/README.md

```diff
@@ -4,7 +4,7 @@ The Polkadot-JS API provides easy-to-use wrappers around JSONRPC calls that flow
 
 The API wrappers provide a standard interface for use -
 
-- A static `.create(<optional WsProvider>)` that returns an API instance when connected, decorated and ready-to use
+- A static `.create(<optional WsProvider>, <optional CustomTypes>)` that returns an API instance when connected, decorated and ready-to use
```

Comment:
There's only one argument to `create`, which is either WsProvider, or ApiOptions. Not sure what's the best way to show it concisely here though

Reply:
Ah, see my comment below. I thought you could pass it as an object if you're passing both.Is that intentional or just an open todo?

Reply:
It's mainly for backward-compatibility.

In the beginning, it was `.create(WsProvider)`. Then, we added ApiOptions so that you could either use the old syntax, or pass `.create(ApiOptions)`, where `ApiOptions = {wsProvider: ..., types: ...}`

Reply:
I would just stick with options documentation. (The provider is only for backwards-compat, no need to document that, I'd rather not have it documented to stop use thereof)

Reply:
So, for clarification: can you do:
`create( { wsProvider: <WsProvider>, types: <CustomTypes> } )` ?

https://github.com/polkadot-js/api/blob/b122f0cde099899df2c5e17018d0d4f1efa13d6c/packages/api/src/types.ts#L96

Reply:
100%